service/src/main/java/bio/terra/cbas/dependencies/github/GitHubService.java

```diff
@@ -0,0 +1,61 @@
+package bio.terra.cbas.dependencies.github;
+
+import com.google.gson.Gson;
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.ClientBuilder;
+import javax.ws.rs.client.WebTarget;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import org.springframework.stereotype.Component;
+
+@Component
+public class GitHubService {
```

Comment:
other dependencies have separate `*Service` and `*Client` classes - why is that? is it relevant here?